README.md

```diff
@@ -23,15 +23,30 @@ Or install it yourself as:
 
 Add the following directive to your Javascript manifest file:
 
-    //= require intlTelInput
+    //= require intl-tel-input-rails
```

Comment:
You don't need to change this or line 30. The file name is `intlTelInput` (same as before).